tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -412,43 +412,81 @@ module.exports = {
     },
     /**
      *
+     * @param {Object.<String,Object>} subSelectors Map of arbitrary attribute name to selector to query
+     * inside the collaborator element, defaults to all when null
      * @returns {Promise.<string[]>} Array of users/groups in share list
      */
-    getCollaboratorsList: async function () {
-      const promiseList = []
+    getCollaboratorsList: async function (subSelectors = null, filterDisplayName = null) {
+      let results = []
+      let informationSelector = {
+        selector: '@collaboratorsInformation',
+        abortOnFailure: false
+      }
+      if (filterDisplayName !== null) {
+        informationSelector = {
+          selector: util.format(this.elements.collaboratorInformationByCollaboratorName.selector, filterDisplayName),
+          locateStrategy: 'xpath',
```

Comment:
better to take the locateStrategy from `this.elements.collaboratorInformationByCollaboratorName.locateStrategy` just for the case if it get changed in the elements section